.gitignore

```diff
@@ -7,3 +7,4 @@ sonar-reports/
 .scannerwork/
 DerivedData/
 build/
+.clang-format
```

Comment:
I use this clang-format

```
BasedOnStyle: Chromium

AlignTrailingComments: true
AllowShortIfStatementsOnASingleLine: false
BreakBeforeBraces: Linux
ColumnLimit: 0
IndentWidth: 4
KeepEmptyLinesAtTheStartOfBlocks: true
Language: ObjC
MaxEmptyLinesToKeep: 4
ObjCSpaceAfterProperty: true
ObjCSpaceBeforeProtocolList: false
PointerBindsToType: false
SpacesBeforeTrailingComments: 1
TabWidth: 4
UseTab: Never
```

Reply:
in your experience, what's the standard in iOS codebases / SDKs you've seen for standardizing code style and linting?

this seems pretty close to an `.eslintrc` that we could check in to keep our code standardized across the company and with open source contributions too

Reply:
yeah, the pretty standard practice, at least AFAIK, is have a .clang-format in the codebase, and everyone has that in git commit / push hook.